pkg/capi/aws.go

```diff
@@ -143,6 +143,81 @@ var _ = Describe("Cluster API AWS MachineSet", framework.LabelCAPI, framework.La
 		Expect(err).ToNot(HaveOccurred(), "Failed to create CAPI machineset")
 		framework.WaitForCAPIMachinesRunning(ctx, cl, machineSet.Name)
 	})
+
+	//OCP-78677 - [CAPI] Dedicated tenancy should be exposed on aws providerspec.
+	It("should be able to run a machine with dedicated instance", func() {
+		awsMachineTemplate = newAWSMachineTemplate(mapiDefaultProviderSpec)
+		awsMachineTemplate.Spec.Template.Spec.Tenancy = "dedicated"
+		Expect(cl.Create(ctx, awsMachineTemplate)).To(Succeed(), "Failed to create awsmachinetemplate")
+		machineSetParams = framework.UpdateCAPIMachineSetName("aws-machineset-78677", machineSetParams)
+		machineSet, err = framework.CreateCAPIMachineSet(ctx, cl, machineSetParams)
+		Expect(err).ToNot(HaveOccurred(), "Failed to create CAPI machineset")
+		framework.WaitForCAPIMachinesRunning(ctx, cl, machineSet.Name)
+	})
+
+	//huliu-OCP-75662 - [CAPI] AWS Machine API Support of more than one block device.
+	It("should be able to run a machine with more than one block device", func() {
+		awsMachineTemplate = newAWSMachineTemplate(mapiDefaultProviderSpec)
+		encryptBool := true
+		unencryptBool := false
+		awsMachineTemplate.Spec.Template.Spec.NonRootVolumes = []awsv1.Volume{
+			{
+				DeviceName: "/dev/xvda",
+				Size:       120,
+				Type:       awsv1.VolumeTypeGP3,
+				IOPS:       5000,
+				Encrypted:  &encryptBool,
+			},
+			{
+				DeviceName: "/dev/sdf",
+				Size:       120,
+				Type:       awsv1.VolumeTypeGP2,
+				Encrypted:  &unencryptBool,
+			},
+		}
+		Expect(cl.Create(ctx, awsMachineTemplate)).To(Succeed(), "Failed to create awsmachinetemplate")
+		machineSetParams = framework.UpdateCAPIMachineSetName("aws-machineset-75662", machineSetParams)
+		machineSet, err = framework.CreateCAPIMachineSet(ctx, cl, machineSetParams)
+		Expect(err).ToNot(HaveOccurred(), "Failed to create CAPI machineset")
+		framework.WaitForCAPIMachinesRunning(ctx, cl, machineSet.Name)
+	})
+
+	//huliu-OCP-75663 - [CAPI] User defined tags can be applied to AWS EC2 Instances.
+	It("should be able to run a machine with user defined tags", func() {
+		awsMachineTemplate = newAWSMachineTemplate(mapiDefaultProviderSpec)
+		awsMachineTemplate.Spec.Template.Spec.AdditionalTags = map[string]string{
+			"adminContact": "qe",
+			"costCenter":   "1981",
+			"customTag":    "test",
+			"Email":        "qe@redhat.com",
+		}
+		Expect(cl.Create(ctx, awsMachineTemplate)).To(Succeed(), "Failed to create awsmachinetemplate")
+		machineSetParams = framework.UpdateCAPIMachineSetName("aws-machineset-75663", machineSetParams)
+		machineSet, err = framework.CreateCAPIMachineSet(ctx, cl, machineSetParams)
+		Expect(err).ToNot(HaveOccurred(), "Failed to create CAPI machineset")
+		framework.WaitForCAPIMachinesRunning(ctx, cl, machineSet.Name)
+	})
+
+	//OCP-76794 - [CAPI] Support AWS capacity-reservations in CAPA.
+	It("should be able to run a machine with capacity-reservations", framework.LabelQEOnly, func() {
```

Comment:
I think this should work on the dev account now, do we want to keep as QE only or run on dev account too?

Reply:
I have updated the capacity reservations to reserve only 35 minutes, so I think it's safe to run on dev account too, updated, thank you!